extras/com.giadamusic.Giada.metainfo.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Giada Developers -->
+<component type="desktop">
+ <id>com.giadamusic.Giada</id>
+ <metadata_license>CC0</metadata_license>
+ <project_license>GPL-2.0+</project_license>
+ <name>Giada</name>
+ <summary>Your hardcore loop machine</summary>
+ <description>
+   <p>
+     Giada is an open source, minimalistic and hardcore music
+     production tool. Designed for DJs, live performers and electronic
+     musicians.
+   </p>
+ </description>
+ <content_rating type="oars-1.1" />
+ <launchable type="desktop-id">com.giadamusic.Giada.desktop</launchable>
+ <screenshots>
+  <screenshot type="default">
+    <image>
+      https://www.giadamusic.com/images/screenshots/giada-loop-machine-screenshot-03-large-project.png
+    </image>
+   <caption>A fairly large project with samples and MIDI events</caption>
+  </screenshot>
+  <screenshot>
+    <image>
+      https://www.giadamusic.com/images/screenshots/giada-loop-machine-screenshot-17-midi-action-editor.png
+    </image>
+   <caption>New Action Editor for MIDI events</caption>
+  </screenshot>
+  <screenshot>
+    <image>
+      https://www.giadamusic.com/images/screenshots/giada-loop-machine-screenshot-01-sample-editor.png
+    </image>
+   <caption>Chopping samples in the advanced Sample Editor</caption>
+  </screenshot>
+ </screenshots>
+ <update_contact>giadaloopmachine_AT_gmail.com</update_contact>
+ <url type="homepage">https://www.giadamusic.com/</url>
+ <url type="help">https://www.giadamusic.com/forum</url>
+ <releases>
```

Comment:
Honestly I would drop the `<releases>` part, not mandatory in the [Freedesktop specs](https://freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-contents-suggestions).  Otherwise it would translate into additional effort on our side to maintain it, on top on an already complicated packaging workflow.

Reply:
`release` are mandotory on flathub whch is the reason I submitted this. So I kept the most recent one now.